x/concentrated-liquidity/swaps.go

```diff
@@ -808,3 +808,120 @@ func edgeCaseInequalityBasedOnSwapStrategy(isZeroForOne bool, nextInitializedTic
 	}
 	return nextInitializedTickSqrtPrice.LT(computedSqrtPrice)
 }
+
```

Comment:
Note to reviewer, `ComputeMaxInAmtGivenMaxTicksCrossed` follows nearly the same logic as calcIn/Out, but tracks the ticks traversed, does not track the spread or uptime accums, and does not save changes to state.